Resources/template.json

```diff
@@ -0,0 +1,297 @@
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      },
+      "cosmosDbCollectionSsaTier1s": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      },
+      "cosmosDbCollectionSsaTier2s": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      },
+      "cosmosDbCollectionStandards": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      },
+      "cosmosDbCollectionStandardsSectorCodes": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      }
+    },
+    "variables": {
+      "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/ArmTemplates/",
+      "functionAppName": "[concat(parameters('functionAppPrefix'),'-fa')]",
+      "appInsightName": "[concat(parameters('functionAppPrefix'),'-ai')]"
+    },
+    "resources": [{
+        "apiVersion": "2017-05-10",
+        "name": "pprefdataAppInsights",
+        "type": "Microsoft.Resources/deployments",
+        "properties": {
+          "mode": "Incremental",
+          "templateLink": {
+            "uri": "[concat(variables('deploymentUrlBase'),'application-insights.json')]",
+            "contentVersion": "1.0.0.0"
+          },
+          "parameters": {
+            "appInsightsName": {
+              "value": "[variables('appInsightName')]"
+            },
+            "attachedService": {
+              "value": "[variables('functionAppName')]"
+            }
+          }
+        }
+      },
+      {
+        "apiVersion": "2017-05-10",
+        "name": "pprefdataAppService",
+        "type": "Microsoft.Resources/deployments",
+        "dependsOn": [
+          "pprefdataAppInsights"
+        ],
+        "properties": {
+          "mode": "Incremental",
+          "templateLink": {
+            "uri": "[concat(variables('deploymentUrlBase'),'app-service.json')]",
+            "contentVersion": "1.0.0.0"
+          },
+          "parameters": {
+            "appServiceName": {
+              "value": "[variables('functionAppName')]"
+            },
+            "appServicePlanName": {
+              "value": "[parameters('appServicePlanName')]"
+            },
+            "appServicePlanResourceGroup": {
+              "value": "[parameters('appServicePlanResourceGroup')]"
+            },
+            "appServiceType": {
+              "value": "functionapp"
+            },
+            "appServiceAppSettings": {
+              "value": [
+                {
+                  "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
+                  "value": "[reference('pprefdataAppInsights').outputs.InstrumentationKey.value]"
```

Comment:
Looks like this was copy and pasted from another project without updating to reflect the name of this project